admin/src/pages/item.vue

```diff
@@ -44,9 +44,44 @@
           <v-col cols="8">
             <base-material-card class="px-5 py-3">
               <template v-slot:heading>
-                <div class="display-2 font-weight-light">
-                  Item {{ item.id }}
-                </div>
+                <v-row no-gutters>
+                  <v-col class="flex-grow-1 display-2 font-weight-light">
+                    Item {{ item.id }}
+                  </v-col>
+                  <v-col
+                    cols="auto"
+                    class="d-flex align-center"
+                  >
+                    <v-menu
+                      offset-y
+                      bottom
+                      left
+                    >
+                      <template v-slot:activator="{ on, attrs }">
+                        <v-btn
+                          v-bind="attrs"
+                          v-on="on"
+                          icon
+                          :loading="isCopyingAttributes"
+                          :disabled="isSaving || isCopyingAttributes"
+                        >
+                          <v-icon>mdi-dots-vertical</v-icon>
+                        </v-btn>
+                      </template>
+                      <v-list>
+                        <v-list-item
+                          v-if="hasParent"
+                          @click="copyAttributesFrom(item.ancestors[0].id)"
+                        >
+                          <v-list-item-title>Copy all attributes from parent</v-list-item-title>
```

Comment:
sorry for misconception, copy from parent button not needed, please make this automatically when creating child item.

Reply:
So, instead of the "Copy from parent" button, we'll use the other way around - a user will create a child using the "Create child" button (that you mentioned in the comment below).

Reply:
not only, same will be when creating child item from media manager.
The idea here is more on server while creating item, if has parent copy attributes.

Reply:
Just to be sure, when a user creates a child item from media manager - we also copy attributes from a parent, right?

Reply:
Correct